### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by Andre601**
I think the readme of this project deserves a bit more information, which is why I create this PR.

It is currently a Draft PR, to allow more discussion and change requests without accidental merging (Although I don't think this would be the case.)

If you have any concerns or suggestions for changes, let me know.

**Current ToDo**
- Implement Spigot URL when plugin gets released (eventually)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+[Wiki]: https://github.com/CodeCrafter47/AdvancedTabOverlay/wiki
 
-File locations
---------------
-- `src/site/markdown`: Markdown files that will be included in the generated site
-- `src/main/java`: Main plugin source files
-- `src/test/java`: Test source files
+# AdvancedTabOverlay
+AdvancedTabOverlay is a tab list plugin for Spigot, which provides highly customisable features including but not limited to:
+- Fake Players (Custom slots)
```
Comment (CodeCrafter47) [inline README.md]: Preferably this would just say _Custom slots_. I don't refer to them as _Fake Players_.

**Comment by CodeCrafter47** [comment]
I plan on releasing the plugin on Spigot soon. I'll mark it as a _beta_ version - so people will know it might crash their server.

**Comment by Andre601** [comment]
Sorry for the delay. I changed it to
```
- Custom slots with icon, text and ping
  - Those can even be animated!
```

**Comment by CodeCrafter47** [comment]
It's here: <https://www.spigotmc.org/resources/advancedtaboverlay.78905/>

File: README.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+AdvancedTabOverlay is a tab list plugin for Spigot, which provides highly customisable features including but not limited to:
+- Custom slots with icon, text and ping
+  - Those can even be animated!
+- Displaying of different tab lists depending on various conditions
+  - Per Player
+  - Per World
+  - Per Group
+  - ...
+- Supports multiple plugins including LuckPerms, Vault, EssentialsX and more.
+- Tons of placeholders to use within your tab lists.
```
Comment (CodeCrafter47) [inline README.md:24]: We could mention PlaceholderAPI support here?

**Review by CodeCrafter47 — APPROVED** [review]
Is there some place where we could add the discord link, or does that make no sense for the readme?

File: README.md (hunk 2 above)
Comment (Andre601) [reply README.md:24]: Added it

**Comment by Andre601** [comment]
I think we could have the link somewhere. Do you have it handy right now?

**Comment by CodeCrafter47** [comment]
<https://discord.gg/qYX5AyJ>

**Comment by Andre601** [comment]
I added the invite to the readme.

**Comment by CodeCrafter47** [comment]
Merged it, thanks!

**Pull request merged**